CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Removed
 - Removed unnecessary `# -*- coding: utf-8 -*-` headers from .py files ([#615](https://github.com/opensearch-project/opensearch-py/pull/615))
 ### Fixed
+- Fixed key error bug which threw an exception when scroll is used and search results are empty because the hits key is not within hits dictionary ([#616](https://github.com/opensearch-project/opensearch-py/pull/616))
```

Comment:
Let's get more specific and less verbose. How about something like "Fix `KeyError` when `scroll` return no `hits`"? We don't need to explain why in CHANGELOG.